src/pages/AppDashboard.vue

```diff
@@ -238,14 +241,21 @@ function goToConfigure() {
 const SmartContractIcon = ref(CopyIcon)
 const smartContractTooltip = ref('Click to copy')
 
-function copyAppId() {
-  SmartContractIcon.value = CheckIcon
-  smartContractTooltip.value = 'Copied'
-  copyToClipboard(appId.value)
-  setTimeout(() => {
-    SmartContractIcon.value = CopyIcon
-    smartContractTooltip.value = 'Click to copy'
-  }, 3000)
+async function copyAppId() {
+  try {
+    SmartContractIcon.value = CheckIcon
+    smartContractTooltip.value = 'Copied'
+    await copyToClipboard(appId.value)
+    toast.success('App ID copied')
+  } catch (e) {
+    console.error(e)
+    toast.error('Failed to copy. Try again or contact support')
+  } finally {
+    setTimeout(() => {
+      SmartContractIcon.value = CopyIcon
+      smartContractTooltip.value = 'Click to copy'
+    }, 3000)
```

Comment:
why 3 seconds?

Reply:
To achieve this effect


https://user-images.githubusercontent.com/24249988/174618444-9f138292-a814-4b40-8220-9c48ec4a2786.mp4